mantidimaging/gui/windows/live_viewer/model.py

```diff
@@ -259,9 +259,8 @@ def _is_image_file(file_name: str) -> bool:
         :param file_name: name of file
         :return: True if file is an image file
         """
-        image_extensions = ['.tif', '.tiff']
-        file_names = any(file_name.lower().endswith(ext) for ext in image_extensions)
-        return file_names
+        image_extensions = ('tif', 'tiff', 'fits')
+        return file_name.rpartition(".")[2].lower() in image_extensions
```

Comment:
Just need to remove `.lower()` from here to meet Sam's micro optimisation suggestion now we are using `.lower` in `presenter.load_and_display_image` and this is good to go 👍

Reply:
This is called from `find_images()` which is passing the original names, so it does need the `lower()`

Reply:
Though as its only called there, I would be tempted to inline it into line 188 as
`if file_path.suffix.lower() in (".tif", ".tiff", ".fits")`
and remove the `_is_image_file()` method.